sqlglot/dialects/prql.py

```diff
@@ -78,6 +79,22 @@ def _parse_selection(self, query: exp.Query, append: bool = True) -> exp.Query:
 
             return query.select(*selects, append=append, copy=False)
 
+        def _parse_take(self, query: exp.Query) -> exp.Query:
+            if self._match(TokenType.NUMBER):
+                first = self._prev.text  # TAKE 4..10 -> self._prev.text = "4."
```

Comment:
This doesn't look right. We probably need to tokenize `1..10` differently. One way is to produce `[1, .., 10]`, i.e. create a `DDOT` token that represents `".."` and then add a method that parses `NUMBER DDOT NUMBER` into a `Slice` or something.